wandb/apis/public.py

```diff
@@ -2136,6 +2087,458 @@ def __repr__(self):
         return "<Run {} ({})>".format("/".join(self.path), self.state)
 
 
+class QueuedRun(Attrs):
```

Comment:
Seems like you are doing a lot of work to proxy method calls to the underlying run object. this is going to be hard to maintain. Could we just have a getter that returns the underlying run object?